includes/class-logger.php

```diff
@@ -68,19 +54,7 @@ public static function init_logger() {
 	 * @return bool Depending on the enable_logging setting.
 	 */
 	public static function can_log() {
-		if ( ! function_exists( 'wc_get_logger' ) ) {
-			return false;
-		}
-
-		if ( is_null( WC_Payments::get_gateway() ) ) {
-			return false;
-		}
-
-		if ( WC_Payments::mode()->is_dev() ) {
-			return true;
-		}
-
-		return 'yes' === WC_Payments::get_gateway()->get_option( 'enable_logging' );
+		return wcpay_get_container()->get( \WCPay\Internal\Logger::class )->can_log();
```

Comment:
Now that we're not using the global `WC_Payments` in this method, there is a `use` statement that we should remove.